lib/miq_expression/field.rb

```diff
@@ -90,6 +94,10 @@ def column_type
     end
   end
 
+  def arel_attribute
```

Comment:
@imtayadeway you ok making this public?

1. It is basically public now with the delegation
2. We want to get away from `arel_attribute`, so feels a little bit in the wrong direction. But maybe I'm wrong here